frontend/components/panels/VaultLiquidationPanel.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+    <BasePanel :current-state="currentStateAndTitle.name">
+        <template #title>{{ currentStateAndTitle.title }}</template>
+        <TextBlock v-if="isExplanationsShown">
+            Successful liquidation of the vault will transfer combined incentive into the wallet that executes the
+            transaction (or a specified wallet). This transaction will start a collateral auction that one can also
+            participate in on a
+            <nuxt-link :to="`/collateral?network=${network}`">separate page</nuxt-link>.
+        </TextBlock>
+        <div class="flex mt-4 justify-end gap-5">
+            <ExecuteWithOtherWalletModal
+                :is-shown.sync="isExecuteToAnotherWalletModalShown"
+                :default-wallet="walletAddress"
+                class="pb-3"
+                @execute="executeWithOtherWallet"
+                @close="isExecuteToAnotherWalletModalShown = false"
+            />
+            <BaseButton :disabled="!isLiquidatable || disabled" @click="isExecuteToAnotherWalletModalShown = true">
```

Comment:
During the `isLiquidating` this button should also be disabled via `isLoading` prop or via `disabled`, otherwise the user can get confused and execute same transaction twice